conf/layer.conf

```diff
@@ -17,7 +17,7 @@ LAYERRECOMMENDS_meta-toradex-security = "\
     toradex-ti-layer \
 "
 
-LAYERSERIES_COMPAT_meta-toradex-security = "scarthgap"
+LAYERSERIES_COMPAT_meta-toradex-security = "scarthgap styhead"
```

Comment:
@EdTheBearded would you please remove `scarthgap` and leave only `styhead`?

Reply:
Sure, already updated the commit.